src/app/app.component.ts

```diff
@@ -17,6 +17,7 @@ export class AppComponent {
 
   active = false
   prepareRoute(outlet: RouterOutlet) {
+    console.log(window.navigator.userAgent.toLowerCase().includes('mobile'))
```

Comment:
Use this value as a global "isMobile" variable and delete unnecessary console.log()